cmsplugin_articles_ai/admin.py

```diff
@@ -18,7 +18,7 @@ class ArticleAttachmentInline(admin.StackedInline):
     fields = ["name", "attachment_file"]
 
 
-class ArticleAdmin(PlaceholderAdminMixin, admin.ModelAdmin):
+class ArticleAdmin(PublisherAdmin):
```

Comment:
Does the `PublisherAdmin` include all the features provided by `PlaceholderAdminMixin` or how was it ok to remove `PlaceholderAdminMixin`?